fisher.fish

```diff
@@ -117,7 +117,7 @@ function $fisher_cmd_name -d "fish plugin manager"
     command mkdir -p {"$fish_path","$fish_config"}/{conf.d,functions,completions} "$fisher_config" "$fisher_cache"
     or return 1
 
-    set -l completions "$fish_config/completions/$fisher_cmd_name.fish"
+    set -l completions "$fish_path/completions/$fisher_cmd_name.fish"
```

Comment:
But, if fisher.fish is itself in `$fish_path/conf.d`, all files it generates should be in `$fish_path` as well. By that logic `$fish_path` should be ignored when installing the `fisher.fish` function. It should be all or none.